contrib/inventory/hyperone.py

```diff
@@ -0,0 +1,215 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from __future__ import print_function
+from time import time
+
+import requests
```

Comment:
Best practice is to separate stdlib imports from third-party libs using one empty line.
stdlib should go first.

Reply:
Going to fix that in next commit.

BTW. There is tool to lint that: https://pypi.org/project/flake8-isort/

Reply:
You're right, but because of complex requirements we likely cannot use it.